geode-core/src/main/java/org/apache/geode/internal/cache/control/PartitionRebalanceDetailsImpl.java

```diff
@@ -47,6 +47,27 @@
   private final transient PartitionedRegion region;
   private long time;
 
+  @Override
```

Comment:
This class on the `develop` branch has a field `numOfMembers`, which does not occur in `support/1.12` branch. I would recommend fix the bug in `develop` branch first. And add `numOfMembers` in the `toString()` method.

Reply:
Will do.

Reply:
I don't see that I need numOfMembers. I will update the toString though.

Reply:
Note, I have moved my changes to be off develop in the other PR.